src/assets/lang/ko.json

```
@@ -1,6 +1,6 @@
 {
   "common": {
-    "name": "COVID Smart NY(COVID 스마트 NY)",
+    "name": "COVID Alert NY(COVID 스마트 NY)",
     "url": "https://CovidAlertNY.health.ny.gov",
     "message": "COVID-19 극복을 위한 Smart NY(스마트 NY)",
```

```suggestion
    "message": "COVID-19 극복을 위한 Alert NY(스마트 NY)",
```
Do this or wait for a translator?